app/_locales/en/messages.json

```diff
@@ -1324,6 +1324,9 @@
   "lockTimeTooGreat": {
     "message": "Lock time is too great"
   },
+  "lowGasWarning": {
+    "message": "Future transactions will queue after this one. This price was last seen was 2 hours ago."
```

Comment:
Should this `2 hours` value be hardcoded?

Reply:
Very good point, I missed this. Let me check how we can get this value.

Reply:
@darkwing : I changed value here to `some time ago`. We will make it dynamic in a following PR.